src/data/navbar.js

```diff
@@ -1,5 +1,4 @@
 import React from 'react'
-import { Link } from 'gatsby'
```

Comment:
I recommend to `import Link from '../components/IntlLink'`, and use along with `linkType` param.